docs/test.ts

```diff
@@ -472,6 +472,26 @@ tracer.appsec.trackUserLoginFailureEvent('user_id', false, meta)
 tracer.appsec.trackCustomEvent('event_name')
 tracer.appsec.trackCustomEvent('event_name', meta)
 
+tracer.appsec.v2.trackUserLoginSuccess("login")
+tracer.appsec.v2.trackUserLoginSuccess("login", user)
+tracer.appsec.v2.trackUserLoginSuccess("login", user, meta)
+tracer.appsec.v2.trackUserLoginSuccess("login", null, meta)
```

Comment:
i'm really not sure i enjoy the v2. here.
It's not really a v2 of the appsec sdk, it's a v2 only for those specific functions. For future breaking change would we put them into this v2 namespace ? should we add new feature to the v2 namespace ?
it just look a bit not well future-proofed you know.

tracer.appsec.trackUserLoginSuccessV2() at least sounds very specific, we made a breaking change on this function only, idk, not super convinced

Reply:
also lol, try running this method with `process.env.DD_TRACING_ENABLED = '0'` :laughing: can you guess what will happen ?